docs/1_how_it_works/1_1_how_it_works.md

```diff
@@ -0,0 +1,51 @@
+# How Finetuner Works
+
+## Contrastive Metric Learning 👓
+
+From the algorithmic perspective,
+**Finetuner** leverage contrastive metric learning approach to improve your model.
+How it works?
```

Comment:
I think "How it works?" is somehow grammatically incorrect, I would phrase it "How does it work?"